src/Renderer/AirspaceRendererSettings.cpp

```diff
@@ -38,27 +38,97 @@ AirspaceRendererSettings::SetDefaults()
 
 #ifdef HAVE_HATCHED_BRUSH
   classes[OTHER].brush = 2;
+  classes[RESTRICT].brush = 3;
+  classes[PROHIBITED].brush = 3;
+  classes[DANGER].brush = 3;
   classes[CLASSA].brush = 3;
   classes[CLASSB].brush = 3;
   classes[CLASSC].brush = 3;
   classes[CLASSD].brush = 3;
+  classes[NOGLIDER].brush = 3;
   classes[CTR].brush = 3;
   classes[WAVE].brush = 2;
   classes[AATASK].brush = 3;
   classes[CLASSE].brush = 3;
   classes[CLASSF].brush = 3;
+  classes[TMZ].brush = 3;
+  classes[CLASSG].brush = 3;
+  classes[MATZ].brush = 3;
   classes[RMZ].brush = 3;
+  classes[UNCLASSIFIED].brush = 3;
+  classes[RESTRICTED].brush = 3;
+  classes[TMA].brush = 3;
+  classes[TRA].brush = 3;
+  classes[TSA].brush = 3;
+  classes[FIR].brush = 3;
+  classes[UIR].brush = 3;
+  classes[ADIZ].brush = 3;
+  classes[ATZ].brush = 3;
+  classes[AWY].brush = 3;
+  classes[MTR].brush = 3;
+  classes[ALERT].brush = 3;
+  classes[WARNING].brush = 3;
+  classes[PROTECTED].brush = 3;
+  classes[HTZ].brush = 3;
+  classes[GLIDING_SECTOR].brush = 3;
+  classes[TRP].brush = 3;
+  classes[TIZ].brush = 3;
+  classes[TIA].brush = 3;
+  classes[MTA].brush = 3;
+  classes[CTA].brush = 3;
+  classes[ACC_SECTOR].brush = 3;
+  classes[AERIAL_SPORTING_RECREATIONAL].brush = 3;
+  classes[OVERFLIGHT_RESTRICTION].brush = 3;
+  classes[MRT].brush = 3;
+  classes[TFR].brush = 3;
+  classes[VFR_SECTOR].brush = 3;
+  classes[AIRSPACECLASSCOUNT].brush = 3;
 #endif
 
   classes[OTHER].SetColors(RGB8_CYAN);
+  classes[RESTRICT].SetColors(RGB8_RED);
+  classes[PROHIBITED].SetColors(RGB8_RED);
   classes[DANGER].SetColors(RGB8_MAGENTA.Darken());
-  classes[MATZ].SetColors(RGB8_MAGENTA.Darken());
+  classes[CLASSA].SetColors(RGB8_MAGENTA.Darken());
+  classes[CLASSB].SetColors(RGB8_MAGENTA.Darken());
   classes[CLASSC].SetColors(RGB8_MAGENTA.Darken());
   classes[CLASSD].SetColors(RGB8_BLUE);
+  classes[NOGLIDER].SetColors(RGB8_ORANGE);
   classes[CTR].SetColors(RGB8_MAGENTA.Darken());
   classes[WAVE].SetColors(RGB8_YELLOW.Darken());
   classes[AATASK].SetColors(HasColors() ? RGB8_YELLOW : RGB8_MAGENTA);
   classes[CLASSE].SetColors(RGB8_GREEN.Darken());
   classes[CLASSF].SetColors(RGB8_GREEN.Darken());
+  classes[TMZ].SetColors(RGB8_MAGENTA.Darken());
+  classes[CLASSG].SetColors(RGB8_MAGENTA.Darken());
+  classes[MATZ].SetColors(RGB8_MAGENTA.Darken());
   classes[RMZ].SetColors(RGB8_MAGENTA.Darken());
+  classes[UNCLASSIFIED].SetColors(RGB8_MAGENTA.Darken());
+  classes[RESTRICTED].SetColors(RGB8_RED); // Assuming RGB8_RED is defined somewhere
```

Comment:
This exists in src/Look/AirspaceLook.cpp

Reply:
Removed false comment.